src/pybind/mgr/orchestrator/module.py

```diff
@@ -129,6 +129,16 @@ def yaml_representer(dumper: 'yaml.SafeDumper', data: 'HostDetails') -> Any:
 yaml.add_representer(HostDetails, HostDetails.yaml_representer)
 
 
+class DaemonFields(enum.Enum):
+    name = 'name'
+    host = 'host'
+    status = 'status'
+    refreshed = 'refreshed'
+    age = 'age'
+    mem_use = 'mem_use'
+    mem_lim = 'mem_lim'
+    image = 'image'
```

Comment:
as mentioned in a past standup meeting, can we add daemon type and service name here? Then we can re-use this enum set later for filtering as well (since we want to cover the current filtering flags in that) and those seem like valid things to want to sort by anyway.

Reply:
@adk3798 I added the service_name and daemon_type fields